### PR TITLE
Upgrade Jupyterlab Extensions from 1.1.x to Jupyterlab 3.x

### PR DESCRIPTION
**Pull request by bluna-tubi**

### CODE REVIEW

File: packages/nteract-data-explorer/src/index.tsx [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { Widget } from '@lumino/widgets';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { Message } from '@lumino/messaging';
-import DataExplorer, { Props } from '@nteract/data-explorer';
+import DataExplorer from '@nteract/data-explorer';
+import {DataProps, Metadata} from '@nteract/data-explorer/lib/utilities/types'
```
Comment (springuper) [inline packages/nteract-data-explorer/src/index.tsx:8]:
```suggestion
import { DataProps, Metadata } from '@nteract/data-explorer/lib/utilities/types'
```

**Review by springuper — APPROVED** [review]
LGTM.

Is it on staging?

File: package.json [hunk 2]
```diff
@@ -12,16 +12,12 @@
...
-    "npm-force-resolutions": "^0.0.3",
     "rimraf": "~3.0.0",
     "ts-jest": "^24.3.0",
     "typescript": "~3.7.3",
     "minimist": ">=1.2.3"
   },
   "engines": {
     "node": ">=10"
-  },
-  "resolutions": {
```
Comment (springuper) [inline package.json:24]: previously there is a type conflict so we add this resolution explicitly, now there is no error in compilation after upgrading these packages, right?

Comment (bluna-tubi) [reply package.json:24]: Correct, this builds out of the box now.

In fact this override was preventing the build from working correctly.

**Comment by bluna-tubi** [comment]
> LGTM.
> 
> Is it on staging?

Yes, this is on staging.

**Pull request merged**